spec/ParseFile.spec.js

```diff
@@ -205,6 +206,29 @@ describe('Parse.File testing', () => {
       notEqual(file.name(), 'hello.txt');
     });
 
+    it('save file with tags', async () => {
+      spyOn(FilesController.prototype, 'createFile').and.callThrough();
+      const file = new Parse.File('hello.txt', data, 'text/plain');
+      file.setTags({ hello: 'world' });
```

Comment:
Can you please refactor this and with `const tags = {hello:world}` and just refer to this variable in the `expect`?  That makes it easier if we have to amend the tags for special keys/values in the future.